examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -87,7 +87,7 @@
      "name": "stderr",
```

Comment:
<p>May be start the section something like (feel free to edit):</p><p> </p><p>"Now, we will serve our trained models on NVIDIA Triton Inference Server (TIS). TIS is  an open-source inference serving software that helps standardize model deployment and execution and delivers fast and scalable AI in production. To serve recommender models on TIS easily, NVIDIA Merlin team designed and developed Merlin Systems library. Merlin Systems provides tools and operators to be able to serve end-to-end recommender systems pipelines on TIS easily."</p><p> </p><p>You can also add a link for Merlin Systems and Triton Inference Server to its GH repo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>change made</p>